.python-version

```diff
@@ -0,0 +1 @@
+cpython-3.13.3-linux-any-any
```

Comment:
I think this should be. 
I think we can probably get away with always using the latest 3.13

```suggestion
cpython-3.13-linux-any-any
```